helm/ngrok-operator/tests/agent-deployment_test.yaml

```diff
@@ -0,0 +1,92 @@
+suite: test agent-deployment
```

Comment:
We already have a file for agent deployment tests https://github.com/ngrok/ngrok-operator/blob/bad8fd8e0e91f30f6ef9ae790a00a968c3709331/helm/ngrok-operator/tests/agent/deployment_test.yaml#L1

Can we move the relevant tests out of here and into that test file?